src/scratch/data_navn/README.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Lærerveiledning - Bruk data: Jente- og guttenavn
+level: 2
+language: nb
+---
+
+# Om oppgaven {.activity}
+I denne oppgaven skal elevene hente data fra SSB og lage en enkel spørrefunksjon for å sjekke om navn er jente- eller guttenavn.
+
+## Oppgaven passer til: {.check}
+ __Fag__: Matematikk, Norsk
+
+__Trinn__: 2.-7. trinn
+
+__Tema__: Statistikk, Databehandling
+
+__Tidsbruk__: Dobbelttime eller mer.
+
+
+## Kompetansemål {.challenge}
+
+- [ ] __Matematikk, 2. årstrinn__: samle, sortere, notere og illustrere data med tellestreker, tabeller og søylediagrammer, og samtale om prosessen og hva illustrasjonene forteller om datamaterialet
+
+- [ ] __Matematikk, 4. årstrinn__: samle, sortere, notere og illustrere data på hensiktsmessige måter med tellestreker, tabeller og søylediagrammer, med og uten digitale verktøy, og samtale om prosess og framstilling
+
+- [ ] __Matematikk, 7. årstrinn__: representere data i tabeller og diagrammer som er framstilt med og uten digitale verktøy, lese og tolke framstillingene og vurdere hvor nyttige de er
+
+- [ ] __Norsk, 4. årstrinn__: søke etter informasjon, skape, lagre og gjenfinne tekster ved hjelp av digitale verktøy
+
+- [ ] __Norsk, 7. årstrinn__: velge ut og vurdere informasjon fra bibliotek og digitale informasjonskanaler
+
+
+## Forslag til læringsmål {.challenge}
+
+- [ ] Elevene kan forklare hva statistikk er.
+
+- [ ] Elevene skal kunne beskrive noen situasjoner hvor det er viktig å bruke statistikk.
+
+- [ ] Elevene skal kunne bruke og formatere ulike filformater.
+
+
+## Forslag til vurderingskriterier {.challenge}
+
+- [ ] Eleven klarer å fullføre oppgaven og finne statistikk på eget navn.
+
+- [ ] Eleven viser høy måloppnåelse ved å finne og bruke annen statistikk.
+
+
+
+## Forutsetninger og utstyr {.challenge}
+- [ ] __Forutsetninger__: Elevene burde ha en grunnleggende forståelse for statistikk og litt erfaring med Scratch. 
+- [ ] __Utstyr__: Datamaskiner med tilgang internett og excel installert (ikke nettbrett). Eventuelt kan elevene bruke Scratch i nettleseren dersom de har en bruker (eller registrerer seg) på scratch.mit.edu/.Elevene kan fint jobbe to og to.
+
+## Fremgangsmåte
+Her kommer tips, erfaring og utfordringer til de ulike stegene i den faktiske oppgaven. [Klikk her for å se oppgaveteksten.](../data_navn/data_navn.html){target=_blank}
+
+
+# _Steg 1: Statistisk Sentralbyrås navnelister_ {.activity}
+- [ ] Her er det et vanlig problem at elevene glemmer å legge inn tittel på Jentenavn og Guttenavn-kolonnene. Da vil ikke programmet fungere.
+
+# Steg 2: Hent navnelistene inn i Scratch {.activity}
+- [ ] Elevene må huske hvor de laster ned filene slik at de finner tilbake til dem.
+- [ ] Her er det et vanlig problem at elevene glemmer å legge inn tittel på Jentenavn og Guttenavn-kolonnene. Da vil ikke programmet fungere.
```

Comment:
samme beskjed som ovenfor? det holder kanskje å si det en gang selv om det gjelder for flere punkt?